docs/release_notes/v0.14.0.rst

```diff
@@ -0,0 +1,54 @@
+New in 0.14.0 (2021-10-04)
+--------------------------
+
+Changes
+~~~~~~~
+- Added fixes to support PyTorch Lightning 1.4 (`#1103`_)
+- Simplified data handling in R tutorials with sceasy and addressed bugs in package installation (`#1122`_).
+- Moved library size distribution computation to model init (`#1123`_)
+- Updated Contribution docs to describe how we backport patches (`#1129`_)
+- Implemented Latent Dirichlet Allocation as a PyroModule (`#1132`_)
+- Restricted PyTorch Lightning version to <1.4 (`#1145`_)
```

Comment:
I believe this one happened in 0.13.1

Reply:
and then maybe I lifted it and put it back

Reply:
Ah ok, I can remove it then.